src/aat/java/uk/gov/hmcts/cmc/claimstore/tests/helpers/CommonOperations.java

```diff
@@ -50,11 +59,25 @@ private Response saveClaim(ClaimData claimData, String userAuthentication, Strin
             .post("/claims/" + userId);
     }
 
-    public void linkDefendant(String claimExternalId, String userAuthentication, String userId) {
+    public void testCasesRetrievalFor(String uriPath) {
```

Comment:
That API is very ambitious. Method just does make generic GET request so it can be used for anything while method claims it is testing case retrieval. In fact it can be used for anything really like PDF generation.